index.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const {
+  cookiesPage, getAllCookies, saveNewCookie, getCookieByNameWithaboutId, deleteCookieByName, patchCookieByName
+} = require('./controllers/cookies')
+const { aboutCookiesByBatchSize, aboutCookiesByType } = require('./controllers/abouts')
+const { getCookiesByTags } = require('./controllers/tags')
+
+const app = express()
+
+app.set('view engine', 'pug')
+app.use(express.static('public'))
+
+app.get('/api', cookiesPage)
```

Comment:
I would expect the documentation to live at `/` rather than the user having to know to navigate to `/api` without access to your code.